Add monitor config (#3959)
pingcap/docs: tiup/tiup-cluster.md

```

The content and format requirements for files under the specified path are as follows:

- The folder specified in the `dashboard_dir` field of `grafana_servers` need to contain full `*.json` files
- The folder specified in the `rule_dir` field of `monitoring_servers` need to contain full `*.rules.yml` files
- For the format of files specified in the `config_file` field of `alertmanager_servers`, refer to [this template](https://github.com/pingcap/tiup/blob/master/templates/config/alertmanager.yml)

When you execute `tiup reload`, TiUP first deletes all old configuration files in the target machine and then uploads the corresponding configuration from the control machine to the corresponding configuration directory of the target machine. Therefore, if you want to modify a particular configuration file, make sure that all configuration files (including the unmodified ones) are in the same directory. For example, to modify Grafana's `tidb.json` file, you need to first copy all the `*.json` files from Grafana's `dashboards` directory to your local directory. Otherwise, other JSON files will be missing from the target machine.

> **Note:**
>

> > If you have configured the `dashboard_dir` field of `grafana_servers`, complete the following operations when executing the `tiup cluster rename` command to rename the cluster:

Reviewer: `when` can be confusing here because users don't know the order of executing this command and the following three steps. I've checked with @lucklove about the order.
```suggestion
> If you have configured the `dashboard_dir` field of `grafana_servers`, complete the following operations after executing the `tiup cluster rename` command to rename the cluster:
```

>

> > 1. `dashboards` in the local directory contains cluster name information. Change the cluster name in the local `dashboards` to the new cluster name.

Author: Agree with your suggested version. 👍🏻  But could I say "Change the cluster name to the new cluster name in the local `dashboards` directory"? Because the next two items of this list start with a verb, so I think it would be better to unify them. @lilin90

> > 1. `dashboards` in the local directory contains cluster name information. Change the cluster name in the local `dashboards` to the new cluster name.

Reviewer: Yes, it would be better to keep them consistent. It would be more friendly to first tell the place and then the action. So I added another suggestion for step 2.

> > 1. `dashboards` in the local directory contains cluster name information. Change the cluster name in the local `dashboards` to the new cluster name.

Author: Get that. Thanks for you suggestion. 😊

> > 2. Change the `datasource` field in `dashboards` to the new cluster name. `datasource` is named after the cluster name.

Reviewer 2: ```suggestion
> 2. Change the `datasource` field in `dashboards` to the new cluster name.  Because `datasource` is named after the cluster name.
```

When you execute `tiup reload`, TiUP first deletes all old configuration files in the target machine and then uploads the corresponding configuration from the control machine to the corresponding configuration directory of the target machine. Therefore, if you want to modify a particular configuration file, make sure that all configuration files (including the unmodified ones) are in the same directory. For example, to modify Grafana's `tidb.json` file, you need to first copy all the `*.json` files from Grafana's `dashboards` directory to your local directory. Otherwise, other JSON files will be missing from the target machine.

> **Note:**
>
> If you have configured the `dashboard_dir` field of `grafana_servers`, complete the following operations after executing the `tiup cluster rename` command to rename the cluster:
>
> 1. `dashboards` in the local directory contains cluster name information. Change the cluster name in the local `dashboards` to the new cluster name.

> > 2. Change the `datasource` field in `dashboards` to the new cluster name, because `datasource` is named after the cluster name.

Reviewer: ```suggestion
> 2. In the local `dashboards` directory, change `datasource` to the new cluster name, because `datasource` is named after the cluster name.
```
